### PR TITLE
Use common prettyname structure + copier update

### PR DESCRIPTION
**Pull request by jl-wynen**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
+<<<<<<< before updating
 # **ess**sans
+||||||| last update
+# Esssans
+=======
+# ESSsans
+>>>>>>> after updating
```
Comment (SimonHeybrock) [inline docs/index.md:7]: conflict

File: requirements/nightly.in [hunk 2]
```diff
@@ -1,11 +1,3 @@
...
 # --- END OF CUSTOM SECTION ---
 # The following was generated by 'tox -e deps', DO NOT EDIT MANUALLY!
-dask
-graphviz
-pythreejs
-plopp @ git+https://github.com/scipp/plopp@main
-sciline @ git+https://github.com/scipp/sciline@main
-scippneutron @ git+https://github.com/scipp/scippneutron@main
-scippnexus @ git+https://github.com/scipp/scippnexus@main
-https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (SimonHeybrock) [inline requirements/nightly.in:11]: Forgot to run `tox -e deps` (or broken merge)?

**Review by SimonHeybrock — CHANGES_REQUESTED** [review]

File: requirements/nightly.in (hunk 2 above)
Comment (jl-wynen) [reply requirements/nightly.in:11]: I ran it but the resulting .txt files only contained updates (no new or moved dependencies) So I decided against committing it in order to keep this PR more focussed.

File: docs/index.md (hunk 1 above)
Comment (jl-wynen) [reply docs/index.md:7]: Ah, I was wondering why the diff was rendered strangely.

File: requirements/nightly.in (hunk 2 above)
Comment (SimonHeybrock) [reply requirements/nightly.in:11]: So how do you plan to fix the problem then?

Comment (jl-wynen) [reply requirements/nightly.in:11]: What problem?

Comment (SimonHeybrock) [reply requirements/nightly.in:11]: The problem that you removed all the nightly dependencies by running `copier update`, somehow?

Comment (jl-wynen) [reply requirements/nightly.in]: This is the `.in` file. The `.txt` file is unchanged. Next time, when someone runs `tox -e deps`, `nightly.in` will be filled back in.
But I restored the file anyway.

**Review by SimonHeybrock — APPROVED** [review]

File: requirements/basetest.in
Comment (SimonHeybrock) [inline requirements/basetest.in:1]: forgot to add the txt file?

Comment (jl-wynen) [reply requirements/basetest.in:1]: It's unused

Comment (SimonHeybrock) [reply requirements/basetest.in:1]: Do you plan to add it to the gitignore then?

Comment (SimonHeybrock) [reply requirements/basetest.in:1]: Also, @YooSunYoung just opened a PR in the `copier_template` that *does* use it, iirc.

**Pull request merged**